maven-sample/pom.xml

```diff
@@ -4,15 +4,44 @@
   <groupId>sample</groupId>
   <artifactId>scala-module-dependency-sample</artifactId>
   <version>1.0-SNAPSHOT</version>
-  <!-- Maven profiles allow you to support both Scala 2.10 and Scala 2.11 with
+  <!-- Maven profiles allow you to support both Scala 2.10, 2.11 and Scala 2.12 with
```

Comment:
I don't know Maven, so I just sort of assume this is right, but just to double check, did you run it?